### ISSUE TITLE
how to save output log?

### ISSUE THREAD

**Issue by MatheusRV** [sharkdp/hyperfine#64]

**Comment by sharkdp** [comment]
Could you please describe this in a bit more detail? Which output do you want to log?

**Comment by MatheusRV** [comment]
Hello

sorry for not providing a description
I'm doing a search between compile time difference between different static site generators. For this I created a docker for each, and inside it I intend to run the hyperfine to measure the compilation time. My problem is the docker when I send the RUN command it displays a fault, I need to get the hyperfine output and save it to a log file.

Any idea how I can provide this?

**Comment by sharkdp** [comment]
Thank you for the clarification.

This has recently been addressed in #59 (and #63). We now have a new option:
```
        --show-output
            Print the stdout and stderr of the benchmark instead of suppressing it. This
            will increase the time it takes for benchmarks to run, so it should only be
            used for debugging purposes or when trying to benchmark output speed.
```

Also, if an error occurs, you would now see:
```
Error: Command terminated with non-zero exit code. Use the '-i'/'--ignore-failure'
option if you want to ignore this. Alternatively, use the '--show-output' option
to debug what went wrong.
```

I have just released [hyperfine v1.1.0](https://github.com/sharkdp/hyperfine/releases/tag/v1.1.0), so you should be able to use this with the latest version.
